tests/unit/amo/components/TestSearchResult.js

```diff
@@ -64,35 +64,44 @@ describe(__filename, () => {
   it('renders the heading', () => {
     const root = render();
 
-    expect(root.find('.SearchResult-link').children()).toIncludeText(
+    expect(root.find('.SearchResult-title').children()).toIncludeText(
       'A search result',
     );
   });
 
-  it('links the heading to the detail page', () => {
+  it('links the card to the detail page', () => {
     const slug = 'some-addon-slug';
     const addon = createInternalAddon({ ...fakeAddon, slug });
 
     const root = render({ addon });
 
-    expect(root.find('.SearchResult-link')).toHaveProp('to', getAddonURL(slug));
+    expect(root.find('.SearchResult-wrapper')).toHaveProp(
```

Comment:
This is good, but I think, because we have logic that creates either a `to` or an `href` attribute, we would also be checking that there is no `href` attribute in this test.